### ISSUE TITLE
question for test.txt file

### ISSUE THREAD

**Issue by zhuyanfang000** [joelthchao/tensorflow-finetune-flickr-style#4]
hi,Thanks for your share first. I want to know what is the test.txt file look like? In this test.txt file, if a label is need for every image?

**Comment by joelthchao** [comment]
Both `train.txt` and `test.txt` look like:
```
path_to_image_1 label_1
path_to_image_2 label_2
...
```

**Comment by zhuyanfang000** [comment]
Thanks for your supply ,I have another question .What device is required to run this program? In AlexNet model there is a parameter group=2, does it mean train it between two GPUs? I have only one PC and no GPU, can I use this program?
I had run it on my equipment and it has error.
Thank you very much!

**Comment by joelthchao** [comment]
`group=2` doesn't mean you need two GPUs. Actually it means two identical sub-network with different weights. It can run no matter you have GPU or not.

**Comment by zhuyanfang000** [comment]
I understand.Thanks a lot.
Best wishes to you!




发自网易邮箱大师
On 08/03/2017 22:20, Joel wrote:

group=2 doesn't mean you need two GPUs. Actually it means two identical sub-network with different weights. It can run no matter you have GPU or not.
